#!/usr/bin/env python
PACKAGE = "chinorobo_ros_usbcam_01"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("H_min", int_t, 0, "H min value", 90, 0, 179)
gen.add("H_max", int_t, 0, "H max value", 160, 0, 179)
gen.add("S_min", int_t, 0, "S min value", 10, 0, 255)
gen.add("S_max", int_t, 0, "S max value", 200, 0, 255)
gen.add("V_min", int_t, 0, "V min value", 0, 0, 255)
gen.add("V_max", int_t, 0, "V max value", 200, 0, 255)

exit(gen.generate(PACKAGE, "chinorobo_ros_usbcam_01", "Parameters"))